index.html

```diff
@@ -67,9 +67,10 @@ <h2 id="title2">These are some of my projects</h2>
         </a></p>
       </div>
     </section>
-     <a target="_blank" href="https://codepen.io/carlos-alfredo-falconi"id="botton_SeeMore"  role="button" aria-label="See More">See More</a>
+    <a target="_blank" href="https://codepen.io/carlos-alfredo-falconi" id="botton_SeeMore" role="button"
+      aria-label="See More">See More</a>
```

Comment:
in the aria-label is always good to add something more explicit like: "Click here to see more of my work in Codepen.io"